one-ui/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -37,13 +37,10 @@ const EntityTiles = (props) => {
                 let mapArtifacts = response.data;
                 mapArtifacts.sort((a, b) => (a.entityType > b.entityType) ? 1 : -1)
                 setEntityArtifacts([...mapArtifacts]);
-              console.log('GET Mapping Artifacts API Called successfully!',response);
             }
           } catch (error) {
               let message = error;
-              console.log('Error while fetching the mappings!', message);
-          } finally {
-            resetSessionTime();
```

Comment:
was deleting finally block intentional?

Reply:
Fixed with recent push. Resolving conversation.

Reply:
No that got deleted by mistake it seems. Thanks for letting me know. Updated the code again. Please review now.